apps/web/src/utils/experienceUtils.tsx

```diff
@@ -913,3 +914,37 @@ export const useExperienceInfo: UseExperienceInfo = (experience) => {
     date: getExperienceDate(experience, intl),
   };
 };
+
+/**
+ * Returns array of organization or similar names pulled from all experiences except personal
+ *
+ * @param experiences SimpleAnyExperience
+ * @return string[]
+ */
+export const organizationSuggestionsFromExperiences = (
+  experiences: SimpleAnyExperience[],
+): string[] => {
+  const experiencesWithoutPersonal = experiences.filter(
+    (exp) => exp?.__typename && exp.__typename !== "PersonalExperience",
+  );
+  const organizationsForAutocomplete = experiencesWithoutPersonal.map((exp) => {
+    if (isAwardExperience(exp)) {
+      return exp.issuedBy;
+    }
+    if (isCommunityExperience(exp)) {
+      return exp.organization;
+    }
+    if (isEducationExperience(exp)) {
+      return exp.institution;
+    }
+    if (isWorkExperience(exp)) {
+      return exp.organization;
+    }
+    return undefined;
+  });
+  const organizationsForAutocompleteFiltered: string[] = unpackMaybes(
+    organizationsForAutocomplete,
+  );
+
+  return organizationsForAutocompleteFiltered;
+};
```

Comment:
I think we will need to ensure the list returned is unique. If I have experiences with the same org, they show up multiple times in the list.

![2025-01-08_08-18](https://github.com/user-attachments/assets/894a8a61-a4c9-4646-bada-c28175a6eb13)

Reply:
Oh! Good spot!
Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/12427/commits/de2f55124bd967bf884efe94cc88afcc0dc2e0a0